programs/bpf/rust/do.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+# TODO #!/usr/bin/env bash
```

Comment:
This is temporary, I'm seeing issues with Rust's `Command::new` and shell script shabangs. Need to investigate further, at the very least I'll revert this before I submit if I haven't figured it out locally and file an issue.